samples/chat/csharp-ooproc/ChatBotIsolated.cs

```diff
@@ -72,7 +72,7 @@ public class CreateChatBotOutput
 
     public class PostResponseOutput
     {
-        [ChatBotPostOutput("{chatId}")]
+        [ChatBotPostOutput("{chatId}", Model = "gpt-3.5-turbo")]
         public ChatBotPostRequest? ChatBotPostRequest { get; set; }
 
```

Comment:
Can we also get this change in for `GetChatState`? That way we return `application/json` content type for both in proc and OOP:
```
[Function(nameof(GetChatState))]
public static async Task<HttpResponseData> GetChatState(
   [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "chats/{chatId}")] HttpRequestData req,
   string chatId,
   [ChatBotQueryInput("{chatId}", TimestampUtc = "{Query.timestampUTC}")] ChatBotState state,
   FunctionContext context)
{
    var response = req.CreateResponse(HttpStatusCode.OK);
    await response.WriteAsJsonAsync(state);
    return response;
}
```

Reply:
We would also need to update `ChatBotState.cs ` and `ChatMessage.cs` within the worker such that each property also has `JsonPropertyName` as a header.

Example:
```
public class ChatBotState
{
    /// <summary>
    /// Gets the ID of the chat bot.
    /// </summary>
    [JsonPropertyName("id")]
    public string? Id { get; set; }

    /// <summary>
    /// Gets if chat bot exists.
    /// </summary>
    [JsonPropertyName("exists")]
    public bool Exists { get; set; }

    /// <summary>
    /// Gets status of chat bot. Options are Uninitialzied, Active, or Expired.
    /// </summary>
    [JsonPropertyName("status")]
    public string? Status { get; set; }

    /// <summary>
    /// Gets timestamp of when chat bot is created.
    /// </summary>
    [JsonPropertyName("createdAt")]
    public DateTime CreatedAt { get; set; }

    /// <summary>
    /// Gets timestamp of when chat bot is last updated.
    /// </summary>
    [JsonPropertyName("lastUpdatedAt")]
    public DateTime LastUpdatedAt { get; set; }

    /// <summary>
    /// Gets number of total messages for chat bot.
    /// </summary>
    [JsonPropertyName("totalMessages")]
    public int TotalMessages { get; set; }

    /// <summary>
    /// Gets number of total tokens for chatbot.
    /// </summary>
    [JsonPropertyName("totalTokens")]
    public int TotalTokens { get; set; }

    /// <summary>
    /// Gets a list of the recent messages from the chatbot.
    /// </summary>
    [JsonPropertyName("recentMessages")]
    public IReadOnlyList<ChatMessage> RecentMessages { get; set; } = Array.Empty<ChatMessage>();
}
```

Reply:
Changes for ChatMessage:
```
public class ChatMessage
{
    /// <summary>
    /// Initializes a new instance of the <see cref="ChatMessage"/> class.
    /// </summary>
    /// <param name="content">The content of the message.</param>
    /// <param name="role">The role of the chat agent.</param>
    public ChatMessage(string content, string role)
    {
        this.Content = content;
        this.Role = role;
    }

    /// <summary>
    /// Gets or sets the content of the message.
    /// </summary>
    [JsonPropertyName("content")]
    public string Content { get; set; }

    /// <summary>
    /// Gets or sets the role of the chat agent.
    /// </summary>
    [JsonPropertyName("role")]
    public string Role { get; set; }
}
```

Reply:
This ensures the E2E test works for dotnet isolated as well to make sure both samples are returning the same status codes and body content as expected

Reply:
@aishwaryabh if you want, you can alternatively create a PR on top of this one to add the updated samples directly. :)

Reply:
made these changes in worker and samples, wondering if we should publish new package or wait for significant changes.

Reply:
I think we can wait; these samples should still work without the worker changes. The worker changes just ensure the E2E test works as expected (since the JSON property gets bound as pascal casing instead of camel casing)

Reply:
done